### PR TITLE
Add the wp wc com extension list command

### PR DESCRIPTION
**Pull request by thilinah**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR adds the command `wp wc com extension list` which displays a list of extensions purchased by the user who connected the site to WCCOM. The result in this command can be used in `wp wc extension install` command to find which extensions can be installed on the site by the user.

Closes # .

### How to test the changes in this Pull Request:

1. Make sure the site is connected to WCCOM via `<your test site root>wp-admin/admin.php?page=wc-addons&section=helper`
2. Run `wp wc com extension list`
3. This should display a list of extensions available for the site.
Screenshot 2022-07-08 at 15 24 13

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [x] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

**Review by rcstr — DISMISSED** [review]
LGTM, tested and works as described 💯

**Review by rcstr — DISMISSED** [review]
💯

File: plugins/woocommerce/changelog/add-cli-com-extension-list-command [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Significance: minor
+Type: add
+Comment: Add the command `wc com extension list` to retrieve and display the extension list for the connected site.
```
Comment (psealock) [inline plugins/woocommerce/changelog/add-cli-com-extension-list-command]: There is an error in the CI here:

```
Changelog entry may only be empty when Significance is "patch".
```

**Review by psealock — COMMENTED** [review]
@thilinah Code freeze for 6.8 is coming up 1600 UTC Thursday, would it be possible to get this reviewed and merged before then?

**Review by Konamiman — DISMISSED** [review]

File: plugins/woocommerce/changelog/add-cli-com-extension-list-command (hunk 1 above)
Comment (Konamiman) [reply plugins/woocommerce/changelog/add-cli-com-extension-list-command]: The formatting of the file was wrong (an empty line is required after "Type", and the "Comment:" prefix in the message isn't necessary). I've fixed it myself.

Comment (thilinah) [reply plugins/woocommerce/changelog/add-cli-com-extension-list-command]: Thank you for fixing this @Konamiman

**Comment by thilinah** [comment]
> @thilinah Code freeze for 6.8 is coming up 1600 UTC Thursday, would it be possible to get this reviewed and merged before then?

Yes, @rcstr or @denho will check the PR.

**Review by Konamiman — APPROVED** [review]

**Comment by thilinah** [comment]
I see this is the last PR to be merged for 6.8. @Konamiman you can go ahead and merge this PR since the changes are done only to the changelog file after the last approval from @rcstr

**Review by denho — APPROVED** [review]

**Review by rcstr — APPROVED** [review]

**Pull request merged**